### PR TITLE
⬆️ Update Dynamic Ui Core

### PR DESCRIPTION
**Pull request by MaFa961**
**Changes:**

1. Update Dynamic Ui core

PR: #1451

---

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.).
- [x] I've rebased the `develop` branch with my branch before finishing this PR.
- [x] I've **summarized all changes** in a list above.
- [x] I've mentioned all **PRs, which relate to this one**.
- [x] I've prefixed my Pull Request title is according to [gitmoji guide](https://gitmoji.carloscuesta.me/).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -110,7 +110,7 @@
     "@process-engine/bpmn-lint_rules": "1.6.0",
     "@process-engine/management_api_client": "4.2.0",
     "@process-engine/process_engine_runtime": "7.4.0-c6f6ddfc-b297",
-    "@process-engine/dynamic_ui_core": "1.4.0",
+    "@process-engine/dynamic_ui_core": "1.4.0-53e3f10c-b29",
```
Comment (roblen45) [inline package.json]: I think you have to release the dynamic_ui_core package first

Comment (ElRaptorus) [reply package.json]: @roblen45 This is the develop version, which gets released through Jenkins automatically, whenever a merge to develop is made.
So this is ok.

We decided some time ago that we will only publish new full versions on release day.
Before the release is done, using the most recent develop versions is the way to go.

**Comment by alexanderkasten** [comment]
@MaFa961 Für was war das Update? Fixt das irgendwas?

**Comment by roblen45** [comment]
> @MaFa961 Für was war das Update? Fixt das irgendwas?

Ja. Dadurch kann man Tab wieder im Dynamic UI verwenden.

**Comment by SteffenKn** [comment]
Ich fände es hilfreich bei solchen Updates in der "Changes"-Section der PR Beschreibung zu beschreiben was gefixt wird (so wie wir das in der Runtime auch tun).

**Comment by ElRaptorus** [comment]
This has been updated on the current release branch.

**Pull request closed without merging**